pyproject.toml

```
@@ -53,3 +60,28 @@ source2classes = "ga4gh.gks.metaschema.scripts.source2classes:cli"
 [build-system]
 requires = ["setuptools>=65.3", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
+
+
+[tool.ruff]
+line-length = 120
+target-version = "py311"
```

We require python >= 3.12
```suggestion
target-version = "py312"
```